packages/truffle-contract/lib/execute.js

```diff
@@ -85,21 +90,21 @@ var execute = {
   /**
    * Executes method as .call and processes optional `defaultBlock` argument.
    * @param  {Function} fn         method
-   * @param  {Object}   methodABI  function ABI segment w/ inputs & outputs keys.
+   * @param  {Object}   methodABI  Function ABI segment w/ inputs & outputs keys.
    * @return {Promise}             Return value of the call.
    */
   call: function(fn, methodABI, address) {
```

Comment:
I now pass the `methodABI` through so that it can be used in `utils.getTxParams`